src/main/java/uk/gov/hmcts/reform/preapi/controllers/MediaServiceController.java

```diff
@@ -88,6 +90,19 @@ public ResponseEntity<LiveEventDTO> getLiveEvents(@PathVariable String liveEvent
         return ResponseEntity.ok(data);
     }
 
+    @GetMapping("/vod")
+    @PreAuthorize("hasAnyRole('ROLE_SUPER_USER', 'ROLE_LEVEL_1', 'ROLE_LEVEL_2', 'ROLE_LEVEL_3', 'ROLE_LEVEL_4')")
+    public ResponseEntity<PlaybackDTO> getVod(@RequestParam UUID recordingId, @RequestParam String userId) {
+        // todo: dont rely on naming convention, link asset name in db
+        var mediaService = mediaServiceBroker.getEnabledMediaService();
+        var assetName = recordingId.toString().replace("-", "") + "_output";
+        var data = mediaService.playAsset(assetName, userId);
+        if (data == null) {
+            throw new NotFoundException("Asset: " + assetName);
+        }
+        return ResponseEntity.ok(data);
```

Comment:
Probably neater to let the service layer throw more specific exceptions and never return null.
```suggestion
        return ResponseEntity.ok(mediaService.playAsset(assetName, userId));
```